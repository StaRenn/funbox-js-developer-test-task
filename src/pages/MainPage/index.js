import React from 'react';
import "./style.css"
import RoutesList from "../../components/RoutesList";
import YandexMaps from "../../components/YandexMaps";

function MainPage() {
    return (
        <div className={"main-page-wrapper"}>
            <RoutesList />
            <YandexMaps/>
        </div>
    );
}

export default MainPage;