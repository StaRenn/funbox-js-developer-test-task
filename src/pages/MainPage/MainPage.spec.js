import {shallow} from "enzyme";
import React from "react";
import MainPage from "./index";

it('expect to render MainPage', () => {
    expect(shallow(<MainPage/>)).toMatchSnapshot();
})