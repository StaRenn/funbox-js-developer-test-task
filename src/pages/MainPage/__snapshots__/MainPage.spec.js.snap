// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render MainPage 1`] = `
<div
  className="main-page-wrapper"
>
  <RoutesList />
  <YandexMaps />
</div>
`;
